### DISCUSSION TITLE
Error Sounds: Consider enabling for release builds where Log Level has been set to Debug

### DISCUSSION THREAD

**Discussion by elliott94** [nvaccess/nvda#15399] [Ideas]
### Is your feature request related to a problem? Please describe.

When running a snapshot build of NVDA, a sound is played if an error is logged to the log file to alert the user; due to the nature of these builds, this is useful in case of regressions within the code. However, these arn't currently output if running release builds.

Could we possibly consider enabling these sounds to be played if a user has their log level set to "Debug"? My thinking was that when attempting to reproduce an issue/bug, the presence of this sound would give a good indication as to whether an error has been logged. It makes sense that this sound isn't played for lower logging levels; less-advanced users may wonder why they're hearing error sounds with no alert or message that explains this, but chances are that if NVDA's log level has been set to "Debug", there's a good reason for this.

**Comment by Brian1Gaff** [comment]
I like this idea a lot, since as a user of some pretty strange old software 
its often  that I need to know whether the software is just not accessible 
at all or  that nvda may have regressed to not support, say an old version, 
so any pointer to the part of the log is welcome when diagnosing.

**Comment by bhavyashah** [comment]
Now that NVDA has an Advanced Settings dialog, I wonder if it would be wiser to have a check box in there to configure the error sound manually? This setting could be enabled by default for development build and disabled by default for release builds. I don't necessarily disagree with enabling the error sound at the Debug level of logging, but I can see a scenario wherein a user is temporarily asked to switch to Debug level or is just fiddling with the log level, and later ends up with an annoying sound that they are unable to connect back to the logging level combo box. I think a dedicated option for the error sound would also allow for greater flexibility.

**Comment by CyrilleB79** [comment]
I agree with @bhavyashah.
There is no need to force the correlation between the log level and the error beep.
I have opened #12672 to discuss this feature. Feel free to add comments there.

@elliott94 would you be happy with a specific feature rather than linking the error sound to the log level?

Personally, I can see some use cases to have the log level set to DEBUG but no beep on error:
* If someone is asked a log by a developer (of NVDA's core or add-on), the person who is interested to be notified of the error is not the one who will take the log.
* My personal usage is to have always the log set to DEBUG in case there is an issue to report. However, sometimes, we do not want to be notified of all errors (e.g. garbage handler, opening Adobe Acrobat, etc.)

**Comment by Adriani90** [comment]
Users can configure if they want to hear error sounds or not in the advanced settings dialog of NVDA since NVDA 2021.3. I am closing this issue and convert it to a discussion in case there is something still open to discuss.
